pkg/apis/aws/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,5 +1,21 @@
 // +build !ignore_autogenerated
 
+/*
+Copyright The Kubernetes Authors.
```

Comment:
Was this generated? Just curious.

Reply:
Yes, it was generated by the sdk when I executed gen k8s.

Concerning the CRD, I forgot adding this file to the PR, done. Thanks!